test/tmisc.c

```diff
@@ -337,9 +337,19 @@ typedef struct {
    See https://nvd.nist.gov/vuln/detail/CVE-2020-10812 */
 #define CVE_2020_10812_FILENAME "cve_2020_10812.h5"
 
-#define MISC38_FILE "type_conversion_path_table_issue.h5"
-#define MISC39_FILE "set_est_link_info.h5"
-#define MISC40_FILE "obj_props_intermediate.h5"
+/* Definitions for misc. test #38 */
+#define MISC38A_FILE     "tmisc38a.h5"
+#define MISC38A_DSETNAME "Fletcher_float_data_be"
+#define MISC38B_FILE     "tmisc38b.h5"
+#define MISC38B_DSETNAME "unusual_datatype"
+#define MISC38C_FILE     "tmisc38b.h5"
```

Comment:
Should probably use "tmisc38c.h5" for this filename just to avoid any test weirdness in the future

Reply:
Looks like it from the test

Reply:
Fixed